### PR TITLE
Create minikube dev env

### PR DESCRIPTION
**Pull request by adrianchiris**

### CODE REVIEW

**Comment by adrianchiris** [comment]
@ykulazhenkov PTAL.

plan is to later extend to support multi-node env.

File: skaffold.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      kustomize:
+        paths:
+          - config/default
+
+    portForward:
+      - resourceType: deployment
+        resourceName: maintenance-operator-controller-manager
+        namespace: maintenance-operator-system
+        port: 56268
+        localPort: 8082
```
Comment (ykulazhenkov) [inline skaffold.yaml:27]: Can you clarify how this portForward is used?

**Review by ykulazhenkov — APPROVED** [review]
LGTm, I added one question

File: skaffold.yaml (hunk 1 above)
Comment (adrianchiris) [reply skaffold.yaml:27]: thats for connecting remote debugger

**Pull request merged**